src/app/buzz.js

```diff
@@ -0,0 +1,48 @@
+import { sessionReducer } from "../session/session";
+import { applyMiddleware, createStore } from "redux";
+import { loadState, saveState } from "../persistence/storePersistence";
+import { throttle } from "lodash";
+import { RESET_STATE, resetState } from "./actions";
+import testUtils from "../testUtils/utils";
+import thunk from "redux-thunk";
+
+const buzzStore = createStore(
+  buzzReducer,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
+  applyMiddleware(thunk)
```

Comment:
Thunk will help us execute async actions but we will have to disable them for testing and replace the execution of the action by the ``RESET_STATE`` action we've used to simulate if the user is logged in or not in the tests we've written for this PR.